src/plugin.js

```diff
@@ -197,17 +208,33 @@ const initPlugin = function(player, options) {
         <div class="vjs-errors-ok-button-container">
           <button class="vjs-errors-ok-button">${this.localize('OK')}</button>
         </div>`;
+    }
+    else {
+      content.innerHTML =
+       `<div class="vjs-errors-content-container">
+        <h2 class="vjs-errors-headline">${this.localize(error.headline)}</h2>
+          <div><b>${this.localize('Error Code')}</b>: ${(error.type || error.code)}</div>
+          ${details}
+        </div>`;
+    }
+
     display.fillWith(content);
     // Get the close button inside the error display
     display.contentEl().firstChild.appendChild(display.getChild('closeButton').el());
     if (player.width() <= 600 || player.height() <= 250) {
       display.addClass('vjs-xs');
     }
 
+    let closeButton = display.el().querySelector('.vjs-close-button');
     let okButton = display.el().querySelector('.vjs-errors-ok-button');
 
+    videojs.on(closeButton, 'click', function() {
```

Comment:
since both the ok button and close button do the same thing, why not pull out the handler to a shared variable?